main/lsp/ExtractVariable.cc

```diff
@@ -1,11 +1,19 @@
 #include "main/lsp/ExtractVariable.h"
+#include "absl/strings/escaping.h"
 #include "ast/treemap/treemap.h"
 #include "common/sort/sort.h"
 
 using namespace std;
 
 namespace sorbet::realmain::lsp {
 
+void logDebugInfo(const std::shared_ptr<spdlog::logger> logger, const core::GlobalState &gs,
+                  const core::Loc selectionLoc, const std::string message) {
+    logger->debug("ExtractToVariable: {}", message);
+    logger->debug("selectionLoc=\"{}\"", selectionLoc.showRaw(gs));
```

Comment:
Ideally we would put all the variables that aren't `source` on one line, makes it easier to manipulate them in splunk queries. the reason why i didn't do it for `source` in the other `Loc.cc` log lines is because i was worried about my log lines getting like sampled if they're too long.